section4/exercises/classMethods.js

```diff
@@ -28,13 +28,27 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
+changeProtein() {
+  this.protein = "carne asada";
+  }
+
+addTopping(toppings) {
+  this.toppings.push(toppings);
+}
+
 };
 
-// ADD CODE
+var caliBurrito = new Burrito ("chicken", "rice", ["guacamole", "french fries", "black beans"]);
+caliBurrito.changeProtein();
+caliBurrito.addTopping("onion");
```

Comment:
And when you call that method with a param, you pass information in like you've done here.